index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <mete name="viewport" contact="with=device-width, initial-scale=1.0">
+        <title>PATHOU'S calculator</title>
+        <link rel="stylesheet" href="style.css"/>
+        <script src="index.js"></script>
+
+    </head>
+    <body>
+        <form>
+            1st Number : <input type="text" id="firstNumber" /><br>
+            2nd Number : <input type="text" id="secondNumber" /><br><br>
+            <input type="button" onClick="addition()" Value="Addition" />
+            <input type="button" onClick="subtraction()" Value="sutraction" />
+            <input type="button" onClick="multiplication()" Value="multiplication" />
```

Comment:
Upon a drop down box or something of that nature your next step would be making an if else or a switch statement.  The switch statement would be the best fit for this project.  Don't forget D.R.Y. Don't repeat yourself.  All four of those functions can be condensed into just one function.